TabLayout/TabLayout.js

```diff
@@ -281,13 +289,16 @@ const TabLayoutBase = kind({
 			const {keyCode, target} = ev;
 			const {anchorTo, collapsed, orientation, 'data-spotlight-id': spotlightId, rtl, type} = props;
 			const popupPanelRef = document.querySelector(`[data-spotlight-id='${spotlightId}'] .${popupTabLayoutComponentCss.panel}`);
+			const tabLayoutContentRef = document.querySelector(`[data-spotlight-id='${spotlightId}'] .${componentCss.content}`);
 
-			if (forwardWithPrevent('onKeyUp', ev, props) && type === 'popup' && is('cancel')(keyCode) && popupPanelRef?.contains(target) && popupPanelRef?.dataset.index === '0') {
-				if (collapsed) {
-					forward('onExpand', ev, props);
+			if (forwardWithPrevent('onKeyUp', ev, props) && is('cancel')(keyCode)) {
+				if ((type === 'popup' && popupPanelRef?.contains(target) && popupPanelRef?.dataset.index === '0') || (type === 'normal' && !props.disableBackKeyNavigation && !Spotlight.getPointerMode() && tabLayoutContentRef?.contains(target))) {
```

Comment:
As of now, LGTM but, 
Let's consider to merge the condition type `popup` and `normanl` after release branch out